pyart/retrieve/qvp.py

```diff
@@ -1,16 +1,47 @@
 ﻿"""
+pyart.retrieve.quasi_vertical_profile
```

Comment:
I don't believe we need this module link anymore due to how the docs generate. We can still keep the "Retrieval of QVPs from a radar object.